CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.3 (2020-02-12)
+
+- Fixed an Angular compiler template checking error on the file attachment component HTML template. [#100](https://github.com/blackbaud/skyux-forms/pull/100)
```

Comment:
Should it be "...component's HTML template." ?

Reply:
Done!